glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -519,6 +520,14 @@ open class GleanInternalAPI internal constructor () {
             return@launch
         }
 
+        if (pingNames.contains("metrics")) {
+            GleanValidation.metricsPingCount.add()
+        }
+
+        if (pingNames.contains("baseline")) {
```

Comment:
Since we also support custom pings that might contain the `metrics` word (or baseline), can we make this an equality?

Reply:
pingNames is a collection, so this is checking as an equality.  It would only match if there was an element "baseline" in the collection.